src/layouts/home.hbs

```diff
@@ -24,12 +24,13 @@
       <article class="doc">
         <div class="actions">
           {{#with (get-quickstart-page 'stable')}}
-            <a href="{{{relativize this.pub.url}}}" class="button primary">Quickstart</a>
+            <a href="#" class="button primary">Quickstart</a>
           {{/with}}
           {{#with (get-overview-page 'stable')}}
-            <a href="{{{relativize this.pub.url}}}" class="button secondary">Overview</a>
+            <a href="#" class="button secondary overview">Overview</a>
```

Comment:
@ggrossetie with the `#` it was not working, did I miss something ?

Reply:
Sorry, I forgot to revert my local changes :(